### ISSUE TITLE
Bewajhah

### ISSUE THREAD

**Issue by DarkRavan** [DarkRavan/Homework#1]
The autosuggest is show his functionality on typing only one character but in the use case description t might be work after typing the two characters in the search box
#1 @DarkRavan 
[ExamCopy.pdf](https://github.com/DarkRavan/Homework/files/7149081/ExamCopy.pdf)

👪 Parent Meta: META | LOGIN: 👜 Salesperson/ Dealer access app #105
◀️ Previous step: USERS: ⚙️ Admin notifies mobile app user of account details #53
▶️ Next step: LEAD: 👜 Salesperson/ Dealer views lead list #107
📷 UI Mockup
Splash and login
OTP
📜 Workflow
Action	Result	Screen
Launches app	See the loader screen	image
Enters mobile number	GET OTP button becomes active once 10 digits are entered	image
The user enters inactive or unregistered mobile #	Sees error message (both shown here for illustration)	image
The user enters valid mobile #	Goes to OTP input screen. Users can click on "Go Back" to return to the mobile number screen	image
The user enters OTP	User sees "Verifying..." spinner	image
OTP was invalid	User sees error message	image
OTP was valid	User directed to lead list view	#107
❗ Logic, Validation and Messages
For icon/ button nav, see UI mockup in presentation mode. All buttons and icons for these screens are hyperlinked.
Mobile number entry
Less than 10-digits: GET OTP button remains inactive
Users cannot enter more than 10 digits
Users cannot enter non-numeric characters
Persistent helper text: You will be able to log in to this app only after your company’s Admin creates your account.
Error messages replace persistent helper text
Unregistered mobile number: Unregistered mobile number. Contact your system administrator.
Deactivated account: Your account has been deactivated by the system administrator.
OTP entry
User cannot type more than 10 digits in the OTP field (this is a max upper limit. OTP will generally be 6 char or less, depending on the OTP generation service used)
No character count is shown for the OTP field (deliberately)
Incorrect, empty, or partial OTP generates the same error message Invalid OTP
OTP SMS message format
315008 is your OTP for signing into the Glowship app.

OTP Time-out
OTP sent via SMS should remain valid for 10 minutes

**Comment by DarkRavan** [comment]
###

**Comment by DarkRavan** [comment]
Kepp your interest in the education

**Comment by DarkRavan** [comment]
Duplicate of #
